packages/devextreme/testing/testcafe/tests/dataGrid/fixedColumns.ts

```diff
@@ -319,3 +322,214 @@ test('Hovering over a row should work correctly after scrolling when there is a
 
   await makeRowsViewTemplatesAsync(DATA_GRID_SELECTOR, 100);
 });
+
+test('Accessibility: Scrollable should always have focusable element when navigate through grid', async (t) => {
```

Comment:
It's better to split this long test scenario into a few little scenarios that check only one thing

(You can focus on some cell immediately by clicking on it)